### PR TITLE
Modularize JS files related to 'structure'

### PR DESCRIPTION
**Pull request by Piyush3079**
This pr contains the structured code of javascript.

This pr is in continuation of Mod_Js_Config_Js, Before reviewing this pr, review and merge Mod_Js_Config_Js and then review this pr.
Before submitting pull request, please check that every commit:

- [ ] Has proper Signed-Off-By
- [ ] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

File: js/src/classes/ErrorReport.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
  */
 import { AJAX } from '../ajax';
 import { PMA_Messages as PMA_messages } from '../variables/export_variables';
-import { PMA_commonParams } from '../variables/common_params';
+import PMA_commonParams from '../variables/common_params';
```
Comment (devenbansod) [inline js/src/classes/ErrorReport.js]: Why is this PMA_commonParams while it is changed at https://github.com/phpmyadmin/phpmyadmin/pull/14524/files#diff-340666e1d3cfc56d0064172a52016d98R10?

**Review by devenbansod — APPROVED** [review]
LGTM.

Just a few minor comments.

**Review by devenbansod — APPROVED** [review]
A lot of methods still have the `PMA_` prefix, but we can tackle that at the end.

**Comment by nightwarrior-xxx** [comment]
@devenbansod I want to work on this issue ```Introduce modern JS features and tools to phpMyAdmin codebase``` . Can you guide me ?

**Pull request merged**